mofacts/client/views/experiment/card.js

```diff
@@ -1588,6 +1587,14 @@ async function showUserFeedback(isCorrect, feedbackMessage, afterAnswerFeedbackC
         feedbackMessage = "<br>Your answer: " + userAnswer + '. ' + feedbackMessage;
       }
     }
+    if(Session.get('curTdfUISettings').displayUserAnswerInIncorrectFeedback && !isCorrect){
+      //prepend the user answer to the feedback message
+      if(singleLineFeedback){
+        feedbackMessage =  "Your answer: " + userAnswer + '. ' + feedbackMessage;
+      } else {
+        feedbackMessage = "<br>Your answer: " + userAnswer + '. ' + feedbackMessage;
+      }
+    }
```

Comment:
OK, so this seems redundant for iscorrect and !iscorrect isn't it doing the same thing?

Reply:
sorry, talkign abotu lines 1585 -1590 and 1592 -1597

Reply:
@imrryr it's supposed to read in 1581 displayUserCorrectFeedback. I changed it

Reply:
Like these changes!